native/src/main/java/io/ballerina/stdlib/grpc/ServerConnectorListener.java

```diff
@@ -53,10 +53,12 @@
     private static final String SERVER_CONNECTOR_GRPC = "grpc";
 
     private final ServicesRegistry servicesRegistry;
+    private Map<String, Long> messageSizeMap;
 
-    public ServerConnectorListener(ServicesRegistry servicesRegistry) {
+    public ServerConnectorListener(ServicesRegistry servicesRegistry, Map<String, Long> messageSizeMap) {
```

Comment:
Why do we need a `Map<String. Long>` here ? Can't we simply use `long` ?

Reply:
There is a possibility of adding `maxOutboundMessageSize` as well in the future. We can include both values in the map